bot/bot.go

```diff
@@ -36,15 +35,21 @@ func HandleMessageEvents(discord *discordgo.Session, msg *discordgo.MessageCreat
 	if msg.Author.ID == discord.State.User.ID {
 		return
 	}
+	if msg.Author.ID != discord.State.User.ID {
```

Comment:
This if statement doesn't make sense after the one on 35, does it? Just remove it and keep the print?